perun-base/src/main/resources/perun-roles.yml

```diff
@@ -6801,6 +6801,27 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getEnrichedBansForVo_int_List<String>_policy:
+    policy_roles:
+      - PERUNOBSERVER:
+      - GROUPADMIN: Vo
```

Comment:
Not sure about GROUPADMIN and GROUPOBSERVER, check with @bodnara which roles we need.